rfc6962-bis.xml

```diff
@@ -378,21 +378,21 @@ following components:
         </t>
 <figure>
   <artwork>
-    enum { x509_entry(0), precert_entry_V2(3), (65535) } LogEntryType;
+   enum { x509_entry(0), precert_entry_V2(3), (65535) } LogEntryType;
 
-    opaque ASN.1Cert&lt;1..2^24-1&gt;;
+   opaque ASN.1Cert&lt;1..2^24-1&gt;;
 
-    struct {
-        ASN.1Cert leaf_certificate;
-        ASN.1Cert certificate_chain&lt;0..2^24-1&gt;;
-    } X509ChainEntry;
+   struct {
+       ASN.1Cert leaf_certificate;
+       ASN.1Cert certificate_chain&lt;0..2^24-1&gt;;
+   } X509ChainEntry;
 
-    opaque CMSPrecert&lt;1..2^24-1&gt;;
+   opaque CMSPrecert&lt;1..2^24-1&gt;;
 
-    struct {
-        CMSPrecert pre_certificate;
-        ASN.1Cert precertificate_chain&lt;0..2^24-1&gt;;
-    } PrecertChainEntryV2;
+   struct {
+       CMSPrecert pre_certificate;
+       ASN.1Cert precertificate_chain&lt;0..2^24-1&gt;;
+   } PrecertChainEntryV2;
```

Comment:
Unless my system is unique, xml2rfc gives a warning saying that it is outdenting this artwork by one space so that it fits.
I was sick of seeing the warning so I manually outdented each.  The warning stops and as far as I can tell the output is identical.